server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -841,8 +844,10 @@ async def _fetch_prs_by_filters(cls,
             filters.append(PullRequest.updated_at >= time_from)
         if updated_min is not None:
             filters.append(PullRequest.updated_at.between(updated_min, updated_max))
-        if pr_blacklist is not None:
-            filters.append(pr_blacklist)
+        if pr_blacklist is not None and len(pr_blacklist) > 0:
```

Comment:
So this should stay a `BinaryExpression`. The reason is performance, the lists can have big lengths and there are cases when I have to call `fetch_prs` twice with the same blacklist. The lengths may be so big that simply constructing the `notin` expression consumes considerable time inside SQLAlchemy. I recommend making the whitelist an optional BinaryExpression, too.